### PR TITLE
release version v1.3.0

### PR DESCRIPTION
**Pull request by 3062**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: reports/1.3.0.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# v1.3.0
+
+# 基准测试
+
++ 使用[sofaload](https://github.com/antJack/sofaload)在本地搭建简单的性能测试
```
Comment (taoyuanyuan) [inline reports/1.3.0.md:5]: 用容器跑的吧？ 这个说明改掉

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## v1.3.0
+
+### Refactoring
+- Moves to consolidated proxy-wasm implementation and enables wazero by default (#2172) [@Crypt Keeper](https://github.com/codefromthecrypt)
+
+### Optimization
+
+- Optimized parsing xds transparent proxy configuration: add pass-through configuration for unrecognized addresses (#2171) [@3062](https://github.com/3062)
```
Comment (taoyuanyuan) [inline CHANGELOG.md:10]: xDS

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+- Optimizing the golangci execution flow in CI testing  (#2166) [@taoyuanyuan](https://github.com/taoyuanyuan) (#2167) [@taoyuanyuan](https://github.com/taoyuanyuan)
+- Adds integrated benchmarks for proxywasm (#2164) [@Crypt Keeper](https://github.com/codefromthecrypt) (#2169) [@Crypt Keeper](https://github.com/codefromthecrypt)
+- Upgrade the minimum version of go supported by mosn to 1.17 (#2160) [@Crypt Keeper](https://github.com/codefromthecrypt)
```
Comment (taoyuanyuan) [inline CHANGELOG.md:13]: MOSN

File: CHANGELOG_ZH.md [hunk 4]
```diff
@@ -1,5 +1,26 @@
...
+## v1.3.0
+
+### 重构
+- 迁移合并 proxy-wasm 的实现，并默认启用wazero (#2172) [@Crypt Keeper](https://github.com/codefromthecrypt)
+
+### 优化
+
+- 优化解析 xds 透明代理配置：增加对未识别地址的透传配置 (#2171) [@3062](https://github.com/3062)
+- 优化 CI 测试中 golangci 执行流程 (#2166) [@taoyuanyuan](https://github.com/taoyuanyuan) (#2167) [@taoyuanyuan](https://github.com/taoyuanyuan)
+- 为 proxywasm 添加集成基准测试 (#2164) [@Crypt Keeper](https://github.com/codefromthecrypt) (#2169) [@Crypt Keeper](https://github.com/codefromthecrypt)
```
Comment (taoyuanyuan) [inline CHANGELOG_ZH.md:12]: Proxy-Wasm

File: CHANGELOG_ZH.md [hunk 5]
```diff
@@ -1,5 +1,26 @@
...
+- 升级 mosn 支持的 go 的最低版本至 1.17 (#2160) [@Crypt Keeper](https://github.com/codefromthecrypt)
+- 改正 README.md 中的一些问题 (#2161) [@liaolinrong](https://github.com/liaolinrong)
+- 新增基准测试 (#2173) [@3062](https://github.com/3062)
+- 重用子集条目以优化分配/使用内存 (#2119) [@dzdx](https://github.com/dzdx) (#2188) [@liwu](https://github.com/chuailiwu)
```
Comment (taoyuanyuan) [inline CHANGELOG_ZH.md:16]: 写清楚谁重用了

File: CHANGELOG.md [hunk 6]
```diff
@@ -7,13 +7,13 @@
...
-- Adds integrated benchmarks for proxywasm (#2164) [@Crypt Keeper](https://github.com/codefromthecrypt) (#2169) [@Crypt Keeper](https://github.com/codefromthecrypt)
-- Upgrade the minimum version of go supported by mosn to 1.17 (#2160) [@Crypt Keeper](https://github.com/codefromthecrypt)
+- Adds integrated benchmarks for Proxy-Wasm (#2164) [@Crypt Keeper](https://github.com/codefromthecrypt) (#2169) [@Crypt Keeper](https://github.com/codefromthecrypt)
+- Upgrade the minimum version of go supported by MOSN to 1.17 (#2160) [@Crypt Keeper](https://github.com/codefromthecrypt)
 - Fix some problems in the README.md (#2161) [@liaolinrong](https://github.com/liaolinrong)
 - add benchmark (#2173) [@3062](https://github.com/3062)
-- reuse subset entry to optimze alloc/inuse memory (#2119) [@dzdx](https://github.com/dzdx) (#2188) [@liwu](https://github.com/chuailiwu)
+- subsetLoadBalancerBuilder reuse subset entry to optimze alloc/inuse memory (#2119) [@dzdx](https://github.com/dzdx) (#2188) [@liwu](https://github.com/chuailiwu)
```
Comment (taoyuanyuan) [inline CHANGELOG.md:16]: 应该说是那个组件，不用细化到函数

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## v1.3.0
+
+### Refactoring
+- Moves to consolidated proxy-wasm implementation and enables wazero by default (#2172) [@Crypt Keeper](https://github.com/codefromthecrypt)
```
Comment (taoyuanyuan) [inline CHANGELOG.md:6]: Proxy-Wasm

File: CHANGELOG.md (hunk 6 above)
Comment (3062) [reply CHANGELOG.md:16]: subsetLoadBalancerBuilder 这个就是组件的名字呀

Comment (taoyuanyuan) [reply CHANGELOG.md:16]: subsetLoadBalancer 才是组件吧，一个负载均衡类型

**Comment by taoyuanyuan** [comment]
加上新合的pr

File: CHANGELOG_ZH.md [hunk 8]
```diff
@@ -1,5 +1,27 @@
...
+
+### 重构
+- 迁移合并 Proxy-Wasm 的实现，并默认启用 wazero (#2172) [@Crypt Keeper](https://github.com/codefromthecrypt)
+
+### 优化
+
+- 优化解析 xDS 透明代理配置：增加对未识别地址的透传配置 (#2171) [@3062](https://github.com/3062)
+- 优化 CI 测试中 golangci 执行流程 (#2166) [@taoyuanyuan](https://github.com/taoyuanyuan) (#2167) [@taoyuanyuan](https://github.com/taoyuanyuan)
+- 为 Proxy-Wasm 添加集成基准测试 (#2164) [@Crypt Keeper](https://github.com/codefromthecrypt) (#2169) [@Crypt Keeper](https://github.com/codefromthecrypt)
+- 升级 MOSN 支持的 go 的最低版本至 1.17 (#2160) [@Crypt Keeper](https://github.com/codefromthecrypt)
```
Comment (taoyuanyuan) [inline CHANGELOG_ZH.md:13]: Go

File: CHANGELOG_ZH.md [hunk 9]
```diff
@@ -1,5 +1,28 @@
...
+- 为 Proxy-Wasm 添加集成基准测试 (#2164) [@Crypt Keeper](https://github.com/codefromthecrypt) (#2169) [@Crypt Keeper](https://github.com/codefromthecrypt)
+- 升级 MOSN 支持的 Go 的最低版本至 1.17 (#2160) [@Crypt Keeper](https://github.com/codefromthecrypt)
+- 改正 README.md 中的一些问题 (#2161) [@liaolinrong](https://github.com/liaolinrong)
+- 新增基准测试 (#2173) [@3062](https://github.com/3062)
+- subsetLoadBalancer 重用子集条目以优化分配/使用内存 (#2119) [@dzdx](https://github.com/dzdx) (#2188) [@liwu](https://github.com/chuailiwu)
+
+### Bug 修复
+
+- 修复 connpool_binging 在连接 upstream timeout 时出现的 panic 问题 (#2180) [@EraserTime](https://github.com/EraserTime)
+- 修复 cluster LB 算法为 LB_ORIGINAL_DST 时 retryTime 是 0 的问题 (#2170) [@3062](https://github.com/3062)
```
Comment (nejisama) [inline CHANGELOG_ZH.md:21]: 看了下这个PR里没有包含对应的测试的，考虑补充一个手动测试的情况？

File: CHANGELOG_ZH.md [hunk 10]
```diff
@@ -1,5 +1,28 @@
...
+- 修复平滑升级失败 (#2129) [@Bryce-Huang](https://github.com/Bryce-huang)
```
Comment (nejisama) [inline CHANGELOG_ZH.md:22]: 这个修复的具体失败场景是什么，考虑补充一个测试说明

File: reports/1.3.0.md (hunk 1 above)
Comment (nejisama) [inline reports/1.3.0.md:5]: 这个是不是应该用 #2173 的方式运行？

Comment (taoyuanyuan) [reply reports/1.3.0.md:5]: 下个版本再用，还需要一些修改才能更通用

**Review by nejisama — APPROVED** [review]
LGTM

**Review by taoyuanyuan — APPROVED** [review]
LGTM

**Pull request merged**